packages/api-explorer/src/form-components/MultiSchemaField.jsx

```diff
@@ -142,13 +142,21 @@ function findEnumOptionsIndex(enumOptions, value) {
  * Note that this is the index and not the value
  */
 function notifySelectionValue(selectedIndex, selectedValue) {
-  const { formData, onChange, options, registry } = this.props;
+  let { formData } = this.props;
+  const { onChange, options, registry } = this.props;
   const { rootSchema } = registry;
   const { discriminatorSchema } = this.state;
 
   if (discriminatorSchema) {
+    // formData might be `undefined` here under some strange circumstances so let's be proactive and create it if we
+    // need to.
+    if (!formData) {
+      formData = {};
```

Comment:
I know we're setting `formData={selectedValue}` down on line 285 so `selectedValue` might be undefined, but this code runs before that does so I'm not sure what's going on here and what is about that spec that makes this `undefined`.